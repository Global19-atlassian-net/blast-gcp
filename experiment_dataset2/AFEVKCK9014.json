{"RID": "AFEVKCK9014", "StartTime": "2018-05-21T17:22:30.538033", "UserID": "193.146.50.254", "blast_params": "{\"evalue\": 10.0, \"filter_string\": \"L;m;\", \"gapped_alignment\": true, \"hitlist_size\": 100, \"penalty\": -2, \"perc_identity\": 0.0, \"reward\": 1, \"strand\": 3, \"word_size\": 28}", "db_tag": "nt_50M.20180502_1", "program": "blastn", "protocol": "1.0", "query_seq": "Bioseq-set ::= {  seq-set {    seq {      id {        local str \"Query_20103\"      },      descr {        user {          type str \"CFastaReader\",          data {            {              label str \"DefLine\",              data str \">\"            }          }        }      },      inst {        repr raw,        mol na,        length 263,        seq-data ncbi2na 'BAB8323C65F53CF8FCF9422AB9EDABAE757ECB9A200CF81843C4C29BC0D1ECF6FBEF8F4C36CF5D2734747FF9F267053C10EDCCFF112352BDAFE021EAFB2CFD9F7DC4'H      }    }  }}", "query_url": "", "top_N_prelim": 100, "top_N_traceback": 500}