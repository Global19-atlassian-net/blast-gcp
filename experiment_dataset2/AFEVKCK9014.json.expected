RID AFEVKCK9014 prelim_search JSON: []
RID AFEVKCK9014 traceback JSON:[]
